olp-cpp-sdk-core/cmake/ios.cmake

```diff
@@ -22,6 +22,14 @@ if(IOS)
         ${NETWORK_IOS_SOURCES}
         ${NETWORK_IOS_INTERNAL_INCLUDES}
     )
+
+    aux_source_directory(${CMAKE_CURRENT_LIST_DIR}/../src/network2/ios NETWORK2_IOS_SOURCES)
+    file(GLOB_RECURSE NETWORK2_IOS_INTERNAL_INCLUDES "*.h" "*.inl")
```

Comment:
same, replace with a list of file names (could be a separate change)

Reply:
Also watch out that i renamed the cmake var to EDGE_SDK_HTTP_SOURCES instead of NETWORK2_IOS_SOURCES